src/test-run/commands/index.js

```diff
@@ -1,25 +1,54 @@
-export const TYPE = {
-    click:     'click',
-    assertion: 'assertion',
-    testDone:  'test-done'
-};
-
-export function createClickCommand () {
-    return {
-        type:      TYPE.click,
-        arguments: {}
-    };
+import TYPE from './type';
+import Assignable from '../../utils/assignable';
+import { ActionSelectorError } from '../../errors/test-run';
+import { ClickOptions } from './options';
+
+// Validators
+function selector (option, val) {
+    if (typeof val !== 'string')
+        throw new ActionSelectorError(val);
```

Comment:
You should pass type, not value here. Value could be non-serializable object with circular ref and then you screwed up.